source/installation.txt

```diff
@@ -18,7 +18,7 @@ Install {+bi-short+} On Premises
 
 .. _installation-guides:
 
-The |bi| can be installed on premises or hosted in MongoDB Atlas. Follow
+The |bi| can be installed on premises or hosted in MongoDB |atlas|. Follow
```

Comment:
I believe this is |service|
```suggestion
The |bi| can be installed on premises or hosted in MongoDB |service|. Follow
```

Reply:
@zach-carr |service| isn't used in this guide. There is an |atlas| substitution in the snooty.toml, but it gives an error. I think this is an issue for BIC (that we don't use a variable for Atlas), but it is out of scope here, so I left it as is for now.